### ISSUE TITLE
Update to Asciidoctor 2 ?

### ISSUE THREAD

**Issue by jstallkamp** [docToolchain/docToolchain#403]
Are there currently any plans to update docToolchain to the new major release of asciidoctor?

**Comment by rdmueller** [comment]
yes, but slowly. there is a 2.x branch where we've already upgraded some versions (but I think asciidoc is still at 1.5.x).

What are the features you miss with the current version?

**Comment by jstallkamp** [comment]
I have to admit that I just recently restarted checking out doctoolchain. (I am happy to see that submodule integration of doctoolchain now also works on windows and noticed that even further improvements are being discussed). 
I just stumbled across the update when working with blockdiag diagram generation via asciidoctor-diagram which allows to add image options (interactive/inline) to the source code block as well, not jsut to generated images (via image directive).

Probably not the most urgent matter, but I thought I'd ask.

**Comment by rdmueller** [comment]
hm. what do you mean by "interactive/inline" - the image options for svg?
I tried to use them when I tried to add zoom to diagrams, but without success.... maybe I used the wrong Asciidoctor version...

**Comment by jstallkamp** [comment]
Basically, I was looking at: https://asciidoctor.org/docs/user-manual/#taming-svgs and failed to employ it with my "[blockdiag]" blocks to get highlighting and text searchability. 

But I now see that, on the other hand, I would loose default "img"-based  features in my browser (show image, save image as.., etc) when embedding the SVG as object.

So I guess I'll just stick to "normal" SVG image integration for now. 

Shall I leave the (more/too generally worded) issue open for general Asciidoctor 2 issues or do you prefer it being closed?

**Comment by mh182** [comment]
Closed with #1105
